src/OrchardCore.Modules/OrchardCore.Html/wwwroot/Scripts/trumbowyg.media.url.js

```diff
@@ -32,8 +35,8 @@
                   trumbowyg.range.insertNode(img);
                 }
 
-                trumbowyg.syncTextarea();
-                $(document).trigger('contentpreview:render');
+                trumbowyg.$c.trigger('tbwchange');
```

Comment:
Is this triggering the preview event correctly?

Reply:
In file `HtmlField-Trumbowyg.Edit.cshtml` from line 58 to 60 is create a hook to call the live preview update on event `tbwchange`.